test/integration/ynEIGEN/AssetRegistry.t.sol

```diff
@@ -82,7 +82,7 @@ contract AssetRegistryTest is ynEigenIntegrationBaseTest {
         uint256 wstethAmount,
         uint256 woethAmount,
         uint256 rethAmount
-        ) public {
+        ) public skipOnHolesky {
```

Comment:
we use both this skipnHolesky and 

vm.skip(_isHolesky());

let's just do one variant to keep it consistent